extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -950,7 +950,9 @@ public function isPageIdMainPage( $pageId ) {
 	 * @return string
 	 */
 	public function getHostName() {
-		return substr( $this->getGlobal( 'Server' ), 7 );
+		preg_match( "^https?:\/\/", $this->getGlobal( 'Server' ), $matches );
+
+		return $matches[1];
```

Comment:
I don't see any grouping in the pattern. How do you get any value in `$matches`?
Please add some unit tests to cover such issues.